composer/modules/web/src/plugins/help/dialogs/about-dialog.jsx

```diff
@@ -65,7 +64,7 @@ class AboutDialog extends React.Component {
                             />
                             <span className='appname'>Composer</span>
                         </div>
-                        <div className="version">v0.96.1.SNAPSHOT</div>
+                        <div className='version'>v0.96.1.SNAPSHOT</div>
```

Comment:
I will change this version with the other files that need to be changed to have the new version.